code/modules/surgery/organs/brain.dm

```diff
@@ -135,3 +141,21 @@
 	if(ishuman(target) && make_cluwne)
 		var/mob/living/carbon/human/H = target
 		H.makeCluwne() //No matter where you go, no matter what you do, you cannot escape
+
+/// TODO DEBUG REMOVE THIS
+/obj/item/melee/energy/sword/saber/decap
```

Comment:
This should be removed before the PR goes in (haha...unless?)

Reply:
Please remove this now that this is no longer in draft

Reply:
removed